catalyst/contrib/dl/callbacks/tests/test_perplexity_callback.py

```diff
@@ -0,0 +1,58 @@
+import torch
+from transformers import AutoModelWithLMHead, AutoTokenizer
+from transformers.data.data_collator import DataCollatorForLanguageModeling
```

Comment:
due to `transformers`  dependency, you need to add this test to https://github.com/catalyst-team/catalyst/blob/master/bin/tests/check_dl_nlp.sh rather then typical tests.
we could not run them correctly with core Catalyst version.

Reply:
:thinking: deal, rechecked :)
need to think about it a bit more, test looks valid in the end

Reply:
`transformers` installed due to this line:
https://github.com/catalyst-team/catalyst/blob/master/.github/workflows/codestyle.yml#L56
as it is in requirements_nlp

Reply:
Only codestyle test run pytest and I need transformers only for tests. I see no problem with dependencies here because callback will also work correct without transformers library. What about to merge it?)